Doc/library/bisect.rst

```diff
@@ -55,8 +55,8 @@ The following functions are provided:
 .. function:: bisect_right(a, x, lo=0, hi=len(a), *, key=None)
               bisect(a, x, lo=0, hi=len(a), *, key=None)
 
-   Similar to :func:`bisect_left`, but returns an insertion point which comes
-   after (to the right of) any existing entries of *x* in *a*.
+   Similar to :py:func:`~bisect.bisect_left`, but returns an insertion point
+   which comes after (to the right of) any existing entries of *x* in *a*.
```

Comment:
I know opinions differ on this, but I personally prefer to keep the diff minimal and preserve git history -- so I'd rather we didn't reflow the paragraph in cases like this. It's okay if a line goes a little bit over 80 chars long!

Reply:
I quite often read on narrow screens so wrapping is useful, but I could try and re-wrap again to make the diff smaller.

Reply:
I've added a new commit to minimise the diff, hopefully better than current!

A

Reply:
Much better, thank you!